apis/constants.ts

```diff
@@ -0,0 +1,10 @@
+const VERSION = "v1";
+const BASE_URL = "http://localhost:3000/api";
+export const API_URL = `${BASE_URL}/${VERSION}`;
```

Comment:
여기 변수명을 이렇게 바꾸는건 어떨까요? 
- `base url` 이 호스트를 가지고있으면 안될것 같아서요! 
```suggestion
const VERSION = "v1";
const BASE_URL = `/api/${VERSION}`
// when add process env, change host to env;
const HOST = "https://localhost:3000";
export const API_URL = `${HOST}${BASE_URL}`

````